packages/odyssey-lint-staged/src/index.js

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+module.exports = {
+  "*": "yarn prettier --write . --ignore-path .gitignore",
```

Comment:
see my not above about `--write` vs `--check`

https://github.com/okta/odyssey/pull/1065/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R22

Reply:
Responded above

Reply:
ok so this must be a feature of yarn I suppose I didn't realize, we can write `yarn <INSERT-BIN-PATH-EXECUTABLE-HERE>` and it will just work which is interesting. This is how we're able to write `yarn lint-staged` above in the precommit hook as well correct?

Reply:
Thats right!